deploy/helm/sumologic/values.yaml

```diff
@@ -122,6 +122,16 @@ sumologic:
     ## uncomment for the debug mode (disables the automatic run of the setup.sh script)
     # debug: true
 
+    monitors:
+      ## If enabled, a pre-install hook will create k8s monitors in Sumo Logic
+      setupEnabled: true
+
+      ## If true, the installed monitors will be disabled by default
+      disabled: false
```

Comment:
This set of configuration variables is a bit misleading in the context of the rest of the Chart. It's customary to have an `enabled:` key which can be used to control the whole feature. To stay consistent with that, I'd do the following:

`enabled: true` - controls the whole feature, creates monitors in setup and enables them
`monitorStatus: enabled` - sets the monitor status, can be used to disable them

Reply:
OK, that makes sense. I'll update that.